client/src/components/heroSection/heroSection.tsx

```diff
@@ -0,0 +1,51 @@
+import CssBaseline from '@mui/material/CssBaseline';
+import Container from '@mui/material/Container';
+import { Box, Typography } from '@mui/material';
+
+import { ThemeProvider } from '@emotion/react';
+import useStyles, { theme } from './heroStyles';
+
+function HeroSection() {
+  const classes = useStyles();
+  return (
+
+    <section className={classes.heroSection}>
+      <Box
+        className={classes.heroImage}
+        component="img"
+        alt=""
```

Comment:
use real `alt` value

Reply:
I think this image is for decoration purposes so there is  no need to add alt  value